tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp

```diff
@@ -33,25 +33,18 @@ inline void tilize_in(
     uint32_t in_num_subblocks,
     uint32_t out_cb_id) {
 
-    // Configure to RowMajor for tilize (similar to add bcast for bias)
-    MATH(( llk_math_pack_sync_init<SYNC>()  ));
-    PACK(( llk_pack_dest_init<SYNC, DstTileFaceLayout::RowMajor, false>()  ));
```

Comment:
You should use the `*_init_short` methods from the compute api, I dont think we should directly use llk in ops.